<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioMixSetting</class>
 <widget class="QWidget" name="AudioMixSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>背景音乐</string>
   </property>
   <widget class="QSlider" name="audioadjust">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>19</y>
      <width>270</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>选择音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="audiopath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>60</y>
      <width>48</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>调节音量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>音效</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择音乐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audiomix1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>音效1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="audiomix2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>音效2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>调节音量</string>
    </property>
   </widget>
   <widget class="QSlider" name="audioadjust2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
